cartographer/io/migrate_serialization_format_main.cc

```diff
@@ -24,6 +24,8 @@ DEFINE_string(
     "Path to the pbstream file that will be migrated to the new version.");
 DEFINE_string(output_pbstream_file, "",
               "Output filename for the migrated pbstream.");
+DEFINE_bool(uses_old_submap, false,
+            "Set if the input pbstream uses the old probability grid format.");
```

Comment:
Can we name this flag more precisely? "old" seems a bit relative. What has changed for the submaps that we could use as distinctive name? Maybe then name the flag something like "has_submaps_with_...".

Reply:
sounds good to me.